README.md

```diff
@@ -1,9 +1,5 @@
----
-label: Welcome
-icon: home
----
-
```

Comment:
I don't think we should be removing this with this change – will break the sidebar here I think.

<img width="1512" alt="Screenshot 2023-03-22 at 1 58 15 PM" src="https://user-images.githubusercontent.com/8602661/227022865-31912ca4-3380-45e3-a271-0b1724028f18.png">